ui/apps/platform/src/Containers/NetworkGraph/deployment/DeploymentSideBar.tsx

```diff
@@ -51,7 +52,8 @@ function DeploymentSideBar({ deploymentId, nodes, edges }: DeploymentSideBarProp
     const numExternalFlows = getNumExternalFlows(nodes, edges, deploymentId);
     const numInternalFlows = getNumInternalFlows(nodes, edges, deploymentId);
     const listenPorts = getListenPorts(nodes, deploymentId);
-    const deploymentPolicyIds = deploymentNode?.data?.policyIds || [];
+    const deploymentPolicyIds =
+        deploymentNode?.data.type === 'DEPLOYMENT' ? deploymentNode?.data?.policyIds : [];
```

Comment:
oh just curious why this was necessary?

Reply:
I had to do that so typescript knew it had the fields